pages/thc/[processRef]/manager-review.tsx

```diff
@@ -0,0 +1,269 @@
+    : [];
+
+  const signatures = useDataSet(
+    Storage.ResidentContext,
+    "signature",
+    tenantIdsPresentForCheckValue
+  );
+
+  const signatureValues = signatures.result ? signatures.result : {};
+
+  const allTenantNames = tenantsValue.map(({ fullName }) => fullName);
+
+  const tenantsPresent = tenantsValue.filter((tenant) =>
+    tenantIdsPresentForCheckValue.includes(tenant.id)
+  );
+
+  const presentTenantNames = tenantsPresent.map(({ fullName }) => fullName);
+
+  const outsidePropertyImages = useDataValue(
+    Storage.ProcessContext,
+    "property",
+    processRef,
+    (values) => (processRef ? values[processRef]?.outside : undefined)
+  );
+
+  const outsidePropertyImageThumbnails = outsidePropertyImages.result
+    ? outsidePropertyImages.result.images.map((image) => (
+        <Thumbnail
+          key={image}
+          src={image}
+          alt={"Thumbnail of an uploaded image"}
+        />
+      ))
+    : [];
+
+  const extraRows = [
+    { key: "Outside property", value: outsidePropertyImageThumbnails },
+    { key: "Completed by", value: officerFullName },
+    { key: "Date completed", value: submittedDateValue },
+    { key: "The visit was", value: visitType },
+    { key: "The visit took place", value: visitTookPlace },
+    { key: "Reason for visit", value: reasonForVisit },
+    {
+      key: "Present for check",
+      value:
+        presentTenantNames.length > 0
+          ? presentTenantNames.join(", ")
+          : "Loading...",
+    },
+  ].filter(({ value }) => value.length > 0);
+
+  return (
+    <MainLayout
+      title={PageTitles.Review}
+      heading="Review Tenancy and Household Check"
+      pausable
+    >
+      <React.Fragment>
+        <TenancySummary
+          details={{
+            address: address.result,
+            tenants: allTenantNames,
+            tenureType: tenureType.result,
+            startDate: tenancyStartDate.result,
+          }}
+          extraRows={extraRows}
+        />
+        <style>{`
+          .mat-tenancy-summary img {
+            margin-right: 2em;
+          }
+        `}</style>
+      </React.Fragment>
+      <IdAndResidencyReviewSection tenants={tenantsWithPresentStatus} />
+      <HouseholdReviewSection />
+      <PropertyInspectionReviewSection />
+      <WellbeingSupportReviewSection />
+      {otherNotesValues.length > 0 && (
+        <>
+          <Heading level={HeadingLevels.H2}>Other Notes</Heading>
+          {otherNotesValues.map((note, index) => (
+            <Paragraph key={index}>{note}</Paragraph>
+          ))}
+        </>
+      )}
+      <Heading level={HeadingLevels.H2}>Declaration</Heading>
+      <Paragraph>
+        I confirm that the information I have given for the purposes of this
+        form is true and accurate. I understand that giving false information
+        may amount to fraud and would put my tenancy at risk with the result
+        that I may lose my home.
+      </Paragraph>
```

Comment:
Is this relevant for a manager review page?

Reply:
It's on the design, I think so it show what the tenant has signed against, but I can double check with Gill if you think it's not so relevant?

Reply:
Nah. Leave it.